packages/storybook8/stories/Concepts/TogetherMode/Docs.mdx

```diff
@@ -0,0 +1,56 @@
+import { Meta, Source } from '@storybook/addon-docs';
+
+<Meta title="Concepts/TogetherMode" />
+
+import PPTLiveSnippetText from '!!raw-loader!./snippets/TogetherMode.snippet.tsx';
+
+# TogetherMode
+
+Together Mode in Teams uses AI to place everyone in a shared virtual background, creating the feeling of being in the same room.
+It’s built to make meetings more engaging and foster stronger connections.
+
+It’s especially effective for training sessions, workshops, or team-building activities.
+A classroom setting, like the one shown below, promotes learning and collaboration, helping participants stay engaged and comfortable in a familiar atmosphere.
+
+<img style={{ width: 'auto', height: 'auto' }} src="images/together-mode/togethermode-scenario_example.png" />
+
+## Supported Scenarios
+
+Together Mode is available only in Teams meetings and group calls. It can be started exclusively by Teams users with the roles of Organizer, Co-organizer, or Presenter.
+ACS (Azure Communication Services) users cannot initiate Together Mode themselves, but they can view the Together Mode stream if it’s started by a Teams user.
+
+<img style={{ width: 'auto', height: 'auto' }} src="images/together-mode/togethermode-not-active-for-acs-user.png" />
+The option to start or switch to Together Mode is disabled because ACS users do not have permission to initiate Together
+Mode.
+
+<img style={{ width: 'auto', height: 'auto' }} src="images/together-mode/togethermode-active-for-acs-user.png" />
+When a Teams user starts Together Mode, the view option becomes enabled, allowing ACS users to switch to the Together
+Mode view.
+
+## Implementing with the VideoGallery
```

Comment:
Similar to my comment above I think we can rename this variable and avoid any reference to pptlive